src/base/utils/Reaction.jl

```diff
@@ -60,7 +60,7 @@ See also: [`get_atoms`](@ref), [`check_duplicate_reaction`](@ref)
 function is_mass_balanced(rxn::Reaction, model::StandardModel)
     atom_balances = Dict{String,Float64}() # float here because stoichiometry is not Int
     for (met, stoich) in rxn.metabolites
-        atoms = get_atoms(model.metabolites[met])
+        atoms = metabolite_formula(model, met)
         isempty(atoms) && continue # ignore blanks
```

Comment:
`isnothing` here

(there's nothing wrong with it being empty, just an empty molecule, carry on nothing to see :D )

Reply:
wow, this was done too late at night... I thought I used isnothing :/